### PR TITLE
posh 0.13.1 (new formula)

### PR DESCRIPTION
**Pull request by dbkaplun**
- [x] Have you followed the [guidelines for contributing](https://github.com/Homebrew/homebrew-core/blob/master/CONTRIBUTING.md)?
- [x] Have you checked that there aren't other open [pull requests](https://github.com/Homebrew/homebrew-core/pulls) for the same formula update/change?
- [x] Have you built your formula locally with `brew install --build-from-source <formula>`, where `<formula>` is the name of the formula you're submitting?
- [x] Does your build pass `brew audit --strict <formula>` (after doing `brew install <formula>`)?

-----

# posh (Policy-compliant ordinary shell) https://salsa.debian.org/clint/posh

### CODE REVIEW

**Review by fxcoudert — APPROVED** [review]

File: Formula/posh.rb [hunk 1]
```diff
@@ -0,0 +1,27 @@
+class Posh < Formula
+  desc "Policy-compliant ordinary shell"
+  homepage "https://salsa.debian.org/clint/posh"
+  url "https://salsa.debian.org/clint/posh/repository/debian/0.13.1/archive.tar.gz"
+  sha256 "c2c10db047294309a109f09e0c76a0cdc33af39563c847b84149b9ac5210f115"
+
+  head "https://salsa.debian.org/clint/posh.git"
```
Comment (ilovezfs) [inline Formula/posh.rb]: I'd leave this out

File: Formula/posh.rb [hunk 2]
```diff
@@ -0,0 +1,27 @@
...
+
+  depends_on "autoconf" => :build
+  depends_on "automake" => :build
+
+  def install
+    system "aclocal"
+    system "autoconf"
+    system "autoheader"
+    system "automake", "--add-missing"
```
Comment (ilovezfs) [inline Formula/posh.rb]: use autoreconf -fiv instead of the separate commands

File: Formula/posh.rb [hunk 3]
```diff
@@ -0,0 +1,27 @@
...
+    system "./configure", "--disable-debug",
```
Comment (ilovezfs) [inline Formula/posh.rb]:
```
configure: WARNING: unrecognized options: --disable-debug
```

**Review by ilovezfs — CHANGES_REQUESTED** [review]

**Comment by dbkaplun** [comment]
@ilovezfs ready for review

cc @woodruffw

**Review by ilovezfs — APPROVED** [review]

**Pull request closed without merging**